client/plots/matrix.controls.js

```diff
@@ -1283,16 +1283,16 @@ export class MatrixControls {
 		//const label = grpDiv.append('label')
 		//label.append('span').html('')
 		const firstGrpWithGeneTw = tg.find(g => g.lst.find(tw => tw.term.type.startsWith('gene')))
-
+		const s = this.parent.config.settings.hierCluster
 		const groups = tg.map((g, index) => {
 			return {
 				index,
 				name: g.name,
 				type: g.type,
 				lst: g.lst.filter(tw => tw.term.type.startsWith('gene')).map(tw => ({ name: tw.term.name })),
+				dataType: s.dataType,
```

Comment:
breaks [matrix](http://localhost:3000/?noheader=1&mass={%22dslabel%22:%22TermdbTest%22,%22genome%22:%22hg38-test%22,%22nav%22:{%22header_mode%22:%22hidden%22},%22plots%22:[{%22chartType%22:%22matrix%22,%22termgroups%22:[{%22lst%22:[{%22term%22:{%22gene%22:%22TP53%22,%22type%22:%22geneVariant%22}},{%22term%22:{%22gene%22:%22AKT1%22,%22type%22:%22geneVariant%22}},{%22id%22:%22efs%22,%22q%22:{%22mode%22:%22continuous%22}},{%22id%22:%22agedx%22,%22q%22:{%22mode%22:%22continuous%22}},{%22id%22:%22diaggrp%22},{%22term%22:{%22gene%22:%22TP53%22,%22type%22:%22geneExpression%22}}]}],%22divideBy%22:{%22id%22:%22sex%22}}]}) as s is undefined. fixing now